ts/features/itwallet/discovery/screens/ItwDiscoveryInfoScreen.tsx

```diff
@@ -29,15 +32,24 @@ export const ItwDiscoveryInfoScreen = ({
   const { isL3 = false } = route.params ?? {};
 
   const machineRef = ItwEidIssuanceMachineContext.useActorRef();
+  const hasNfcFeature = useIOSelector(itwHasNfcFeatureSelector);
 
   useFocusEffect(
     useCallback(() => {
-      trackItWalletIntroScreen();
-    }, [])
+      if (!isL3 || hasNfcFeature) {
+        trackItWalletIntroScreen();
+      }
+    }, [hasNfcFeature, isL3])
   );
 
   useOnFirstRender(() => {
-    machineRef.send({ type: "start", isL3 });
+    if (!isL3) {
```

Comment:
Maybe here it's safer to use `else if` or an early return. If it's not L3 and the user has `hasNfcFeature` is true both the `if` cases will be run.